vnext/Microsoft.ReactNative/Fabric/FabricUIManagerModule.cpp

```diff
@@ -121,6 +122,13 @@ const IComponentViewRegistry &FabricUIManager::GetViewRegistry() const noexcept
   return m_registry;
 }
 
+int64_t FabricUIManager::getRootSurfaceTag() noexcept {
```

Comment:
I'd really rather not have this API.  Its just asking people to make assumptions about there only being a single surface - which isn't true.

Lets see if we can avoid creating this.  (See comments above about ReactNativeIslands rootTag maybe being something else.